updater.go

```diff
@@ -129,6 +129,8 @@ func (u *Updater) update(ctx Context, options UpdateOptions) (*Update, error) {
 		return update, promptErr(fmt.Errorf("Unknown prompt error"))
 	case UpdateActionContinue:
 		// Continue
+	case UpdateActionUIBusy:
+		// Continue (no need to report this to the server)
```

Comment:
I think the point of finding out `UIBusy` is to _not_ do the update yet. So this should be more like `case UpdateActionError`, right?